##left hand position limit
R_THUMB_JOINT0:
  ID: 1
  Min_Pos: -120.0
  Max_Pos: 5
R_THUMB_JOINT1:
  ID: 2
  Min_Pos: -90.0
  Max_Pos: 120.0
R_INDEX_JOINT0:
  ID: 3
  Min_Pos: 0.0
  Max_Pos: 120.0
R_INDEX_JOINT1:
  ID: 4
  Min_Pos: -90.0
  Max_Pos: 100.0
R_MIDDLE_JOINT0:
  ID: 5
  Min_Pos: 5.0
  Max_Pos: 120.0
R_LOCK_JOINT0:
  ID: 6
  Min_Pos: -1.0
  Max_Pos: 14.0
